docs/components/console/manage-clusters/cluster-backups.md

```diff
@@ -0,0 +1,45 @@
+---
+id: create-backups
+title: Create backup
+description: "If your organization works within Camunda's Enterprise plan, you can create cluster backups."
+---
+
+If your organization works within Camunda's [Enterprise](https://camunda.com/enterprise/) plan, you can create a manual and scheduled backups of your cluster.
```

Comment:
Not sure if you want/ need to link to the backups docs from the guides as well or not needed: https://docs.camunda.io/docs/next/components/concepts/backups/

Reply:
Yes! Always link! I will add it.